components/channel_info_modal/index.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import ChannelInfoModal from './channel_info_modal.jsx';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps
+    };
+}
+
+export default connect(mapStateToProps)(ChannelInfoModal);
```

Comment:
You actually don't need this whole file because it's not adding any more props that aren't being passed in by a parent

Reply:
Actually, I believe I'll still need it, because I just realized that the component is still using the `TeamStore`, so I'll move that out into this file.